vars/patchBuildConfigOutputLabels.txt

```diff
@@ -0,0 +1,29 @@
+# patchBuildConfigOutputLabels
+
+Patches the build config to add jenkins and git build information.
+
+Parameters:
+
+patchBuildConfigOutputLabes(bcName: String (required), domainPrefix: String (optional))
```

Comment:
Would make sense to point out the default value for `domainPrefix` here?
Something like `domainPrefix: String (optional, default: com.redhat)`

Reply:
ok. I made the update. Also fixed my typo in the name.